### PR TITLE
completed typescript-functions-practice

### PR DESCRIPTION
**Pull request by mahdiehmaleki**

### CODE REVIEW

File: typescript-functions-practice/get-first-element.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* exported getFirstElement */
+function getFirstElement(array: number[]): number {
```
Comment (hesam) [inline typescript-functions-practice/get-first-element.ts:2]: array type could be general: unknown[] or any[]

File: typescript-functions-practice/get-last-element.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/* exported getLastElement */
+function getLastElement(array: number[]): number {
```
Comment (hesam) [inline typescript-functions-practice/get-last-element.ts:2]: array: any[] or unknown[]

**Review by hesam — APPROVED** [review]
Awesome!

- Please don't forget to include screenshot of index.html/console page next time
- Please see couple of comments in code about type annotations

**Comment by mahdiehmaleki** [comment]
Sure! Thank you very much.

**Pull request merged**